pkgs/top-level/all-packages.nix

```diff
@@ -41298,6 +41298,8 @@ with pkgs;
 
   utf8cpp = callPackage ../development/libraries/utf8cpp { };
 
+  utf8cpp_4 = callPackage ../development/libraries/utf8cpp/4.nix { };
+
```

Comment:
```suggestion
```

New top-level packages should go to `pkgs/by-name`  https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/README.md

Reply:
Actually it's not a new package, since `utf8cpp` is defined in `all-packages.nix`. Maybe we could move all of them to `pkg/by-name` later.